component/libfuse/libfuse2_handler.go

```diff
@@ -714,6 +714,10 @@ func libfuse2_rename(src *C.char, dst *C.char) C.int {
 	srcPath = common.NormalizeObjectName(srcPath)
 	dstPath := trimFusePath(dst)
 	dstPath = common.NormalizeObjectName(dstPath)
+	if strings.Contains(dstPath, ".fuse_hidden") {
```

Comment:
why is this file getting created. We shall stop it from creation otherwise we are uploading extra data to container which is not intended by user. There is a list of ignoreFiles in libfuse component we shall add this name/pattern there.

Reply:
The ignorefiles is more of a word for word matching, this is a if it starts with this pattern, ignore